src/main.js

```diff
@@ -70,13 +73,13 @@ const addresses = arrayFlatten(
     // eslint-disable-next-line no-unused-vars
     ([name, nic]: [string, mixed]): Array<string> =>
       (nic: any)
-        .filter(
-          (address: Object): boolean =>
-            address.family === 'IPv4' && address.address !== '127.0.0.1',
-        )
+        .filter((address: Object): boolean => {
+          logger.debug({ found: address }, 'Network Interface');
```

Comment:
Based on my expceriences in virtual machines and docker, the information which network interfaces are available is sometime usefull, especially if you select one to bind the server to.

But i will remove this log, cause this is only informational (and is not! used to bind the server to this interface) 
-> I will remove the info.
 
done